composables/usePrefix.ts

```diff
@@ -10,57 +10,35 @@ const sharedPrefix = ref<Prefix>()
 export default function () {
   const route = useRoute()
   const storage = useLocalStorage('urlPrefix', { selected: DEFAULT_PREFIX })
-  const initialPrefixFromPath = getAvailablePrefix(route.path.split('/')[1])
-  const validPrefixFromRoute = computed(() =>
-    getAvailablePrefix(route.params.prefix)
-  )
 
-  const prefix = computed<Prefix>(
+  const urlPrefix = computed<Prefix>(
     () =>
       (sharedPrefix.value ||
-        validPrefixFromRoute.value ||
+        getAvailablePrefix(route.params.prefix) ||
         storage.value.selected ||
```

Comment:
IMO this is less readable than before

Reply:
instead of 3 computedRefs, 
we only need 1